ycast/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.1.0'
+__version__ = '1.1.1'
```

Comment:
Please do not bump the version yourself.

Reply:
Ok